build.sbt

```diff
@@ -322,7 +324,7 @@ lazy val storehausTesting = Project(
   settings = sharedSettings ++ Seq(
     name := "storehaus-testing",
     libraryDependencies ++= Seq(
-      "org.scalacheck" %% "scalacheck" % "1.12.2" withSources(),
+      "org.scalacheck" %% "scalacheck" % "1.12.6" withSources(),
```

Comment:
can we go to `1.13` branch? Also, while we are at it, can we move this version up to the top of the file where we set the other versions?